src/memory/Memory.java

```diff
@@ -39,52 +39,23 @@
 import javax.swing.JOptionPane;
 import javax.swing.JPanel;
 import javax.swing.border.BevelBorder;
-
-/**
- *
- * @author marco
- */
 public class Memory {
-    private JFrame frame;
-    private JPanel panelTitle, panelGrid, panelControl;
-    private JButton buttonNew, buttonSolve, buttonAbout;
-
-    private buttonGame buttonLastClicked;
-    private final Images imagens;
-    
-    private JLabel labelTitle;
-    
-    Integer intQtdOpened;
-    Integer intCombined;
-    ArrayList  listShuffle;
-
-    
-    private List<buttonGame> listButtons;
-    
-    private class buttonGame extends JButton{
-        Integer iCod;
-        public buttonGame(Integer iCod){
-            this.iCod = iCod;
-        }
+    public Memory(Board board) {
     }
-    
-    private void enlargeFont(java.awt.Component c, float factor) {
-        c.setFont(c.getFont().deriveFont(c.getFont().getSize() * factor));
-    }
-    
-    public Memory(){ 
-        imagens = new Images();
+<<<<<<< HEAD
```

Comment:
```suggestion
<<<<<<< HEAD
```
Resolve merge conflicts